### DISCUSSION TITLE
jx:image tag has a scale error.

### DISCUSSION THREAD

**Discussion by akaLogan** [jxlsteam/jxls#208] [Q&A] [answered]
### When I using the **jx:image** tag like this-> **' jx:image(lastCell="xxx" src="xxx" scaleX="1.0" scaleY="1.0") '**, threw a exception that message like this: 
<img width="737" alt="bf61df2854a9ec1705eb00d244dcad2" src="https://user-images.githubusercontent.com/45533843/176725622-e4b0a6ed-e439-42a3-925a-a4aaf28bfb07.png">
### That is a conflict with the 'addImage' method in PoiTransformer.java and caused exception like I said.
<img width="583" alt="0c9c552dc7d10c91e7fce4119f04e1f" src="https://user-images.githubusercontent.com/45533843/176735245-bb66ac55-fb80-405f-a34f-e272d178d2f9.png">
<img width="556" alt="85a7d6db25ca132822790fcf97ad495" src="https://user-images.githubusercontent.com/45533843/176735632-10eeb30c-2c28-44c6-959d-b280f135aff4.png">

**Comment by SoltauFintel** [comment, marked as answer]
Looks to me like an Apache POI XLS bug that has to be reported by you at the POI community.

I guess it will work if you use XLSX.

XLS = HSSF = old format, XLSX = XSSF

**Reply by akaLogan** [reply]
**Thank for help, it was worked. But there still some problem, the picture has been resized to same as first cell size. How can I do to keep picture original size?** @SoltauFintel

**Reply by akaLogan** [reply]
Hello @SoltauFintel , You still here?

**Reply by SoltauFintel** [reply]
I have no experience with JXLS and images. So you'd have to wait for someone else to reply or do your own research. :-)

**Reply by akaLogan** [reply]
All right, Thanks
